wsidicomizer/common.py

```diff
@@ -62,6 +62,10 @@ def samples_per_pixel(self) -> int:
     @property
     def photometric_interpretation(self) -> str:
         # Should be derived from the used subsample format
+        # DICOM 2022a part 3 IODs - C.8.12.4.1.5 Photometric Interpretation and Samples Per Pixel
```

Comment:
Please keep lines to 79 characters.

Reply:
No problem. Simply using the default on PyCharm.